spec/examples/hourly_rule_spec.rb

```diff
@@ -42,10 +42,10 @@ module IceCube
         schedule = Schedule.new(t0 = Time.local(2013, 11, 3, 0, 0, 0))
         schedule.add_recurrence_rule Rule.hourly
         schedule.first(4).should == [
-          Time.local(2013, 11, 3, 0, 0, 0),             # -0700
-          Time.local(2013, 11, 3, 1, 0, 0) - ONE_HOUR,  # -0700
-          Time.local(2013, 11, 3, 1, 0, 0),             # -0800
-          Time.local(2013, 11, 3, 2, 0, 0),             # -0800
+          Time.new(2013, 11, 3, 0, 0, 0, '-07:00'),
```

Comment:
Time.local and Time.new are the same thing: `local` is just a little more explicit so we prefer it when we don't mean `utc`.

Yes, it's the second hour. I think it makes sense since most of Nov 3 is Standard Time, and if you want to specify an hour of that day it should be the expected offset. Only 00:00 to 00:59 are DST.

The `- ONE_HOUR` is a trick to "rewind" from 1AM Standard Time into 1AM DST because there's no other way to specify that "extra" hour literally. (I guess there's `Time.at unixtime` too, though.)